src/views/MentorMe.js

```diff
@@ -2,15 +2,29 @@ import React, { Component } from "react";
 import Signup from "../components/Signup";
 import UserProfile from "../components/UserProfile";
 import Login from "../components/Login";
-import { Route } from "react-router-dom";
+import { Route, Redirect } from "react-router-dom";
 
 export default class MentorMe extends Component {
+//   constructor(props) {
+//     super(props);
+//   }
+
   render() {
     return (
       <div>
-        <Route path="/signup" component={Signup} />
+        <Route
+          exact
+          path="/"
+          render={() =>
+            localStorage.getItem("token") ? (
+                <UserProfile />
+            ) : (
+              <Redirect to="/signup" />
```

Comment:
We need to implement a page between this step and login/signup to give the user the opportunity to choose,

Reply:
There is a nav link if a user already has an account